inference-engine/tests/functional/plugin/gna/shared_tests_instances/transpose_add.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include "subgraph_tests/transpose_add.hpp"
+#include "common_test_utils/test_constants.hpp"
+namespace {
+std::vector<std::vector<size_t>> input_shapes {
+    {1, 4, 32},
+    {1, 8, 8}
```

Comment:
consider to add more tests (with `dim % 4 != 0` ), e.g., {1, 7, 9}, {1, 34, 5}

Reply:
2 additional tests are added, but one of dimensions should be multiple by 8, otherwise the transpose won't be supported by GNA